packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -108,31 +124,17 @@ class _CodeViewState extends State<CodeView>
     }
 
     addAutoDisposeListener(
-      widget.controller.scriptLocation,
+      widget.codeViewController.scriptLocation,
       _handleScriptLocationChanged,
     );
   }
 
-  @override
```

Comment:
why did didUpdateWidget go away?

Reply:
I had moved it at some point, but then changed my mind. Guess I forgot to add it back.